### PR TITLE
feat(presto-client): include get query metadata method

### LINKED ISSUE #12: Include query metadata function
The client should include a method to return any specific query metadata by its ID.

### PR DESCRIPTION
**Pull request by alonsovb**
## Changes

Resolves #12

- Add `getQueryInfo` method to retrieve all information from a given Query ID.
- Include some interfaces for the resulting query info, based on the Java classes from the `presto-main` and `presto-spi` packages from the main PrestoDB repo.
- Refactored the types into a folder now that there's a bit more.

**Note**: Will update README and sample NestJs app soon.

### CODE REVIEW

**Review by jorgeramirezamora — APPROVED** [review]
LGTM!

File: presto-client/README.md [hunk 1]
```diff
@@ -111,7 +128,7 @@ The `getCatalogs` method retrieves all available database catalogs, returning th
 ```typescript
 const catalogs = await prestoClient.getCatalogs()
 console.log(catalogs)
-```
+````
```
Comment (yhwang) [inline presto-client/README.md]: add this by accident I guess

**Review by yhwang — APPROVED** [review]
Only one comment for the README doc. the rest of the changes LGTM. Nice work!

File: presto-client/README.md (hunk 1 above)
Comment (alonsovb) [reply presto-client/README.md]: yes, I'll remove it

Comment (alonsovb) [reply presto-client/README.md]: it's fixed in the latest commit

**Review by yhwang — APPROVED** [review]

**Review by jorgeramirezamora — APPROVED** [review]

**Pull request merged**